kotlin/internal/jvm/jvm.bzl

```diff
@@ -243,19 +243,21 @@ _common_attr = utils.add_dicts(
 
 _lib_common_attr = utils.add_dicts(_common_attr, {
     "exports": attr.label_list(
-        doc = """Exported libraries.
+        doc = """\
```

Comment:
You're astute :)
It just escapes the newline so the documentation still starts on the first line of the string.

It's not needed here. HOWEVER if I make a change upstream so that stardoc automatically dedents the string to the indent of the first line, then this would allow us to align the docstrings in the original way, just adding the whitespace on the first line would cause it to align correctly.

BTW here's a preview of this PR https://stupefied-newton-9c1944.netlify.app/kotlin.html

Reply:
Nice.